pkg/plugins/ca/provided/config/provided_ca_config.pb.validate.go

```diff
@@ -0,0 +1,125 @@
+// Code generated by protoc-gen-validate. DO NOT EDIT.
```

Comment:
Do we want to add new `*.pb.validate.go` files?

Reply:
Can we just add them for now? I'm tired of removing those files.
We should turn off protoc validate generation but it's out of scope of this PR.

Can you create an issue if you feel like we should do something about those files?

Reply:
I don't mind to leave them since they are automatically generated and doesn't require anything from us. Just wanted to know general position for that question